tests/api_resources/PortingOrderTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Telnyx;
+
+/**
+ * @internal
+ * @covers \Telnyx\PortingOrder
+ */
+final class PortingOrderTest extends \Telnyx\TestCase
+{
+    const TEST_RESOURCE_ID = 'f1486bae-f067-460c-ad43-73a92848f902';
+    
+    public function testIsListable()
+    {
+        $this->expectsRequest(
+            'get',
+            '/v2/porting_orders'
+        );
+        $resources = PortingOrder::all();
+        $this->assertInstanceOf(\Telnyx\Collection::class, $resources);
+        $this->assertInstanceOf(\Telnyx\PortingOrder::class, $resources['data'][0]);
+    }
+
+    public function testIsCreatable()
+    {
+        $this->expectsRequest(
+            'post',
+            '/v2/porting_orders'
+        );
+        $resource = PortingOrder::create(["phone_numbers" => ["+13035550000","+13035550001","+13035550002"]]);
+        $this->assertInstanceOf(\Telnyx\PortingOrder::class, $resource);
```

Comment:
@weroh the request is a list of phone numbers and the response payload is a list of results. In this case, you should assert that the first object in the list is a `\Telnyx\PortingOrder`

Reply:
This is now expecting a collection and checking if the first object is a \Telnyx\PortingOrder